www/src/routes/student/resumeReview/UploadResume.tsx

```diff
@@ -0,0 +1,131 @@
+import { useAuth0 } from '@auth0/auth0-react';
+import { Backdrop, CircularProgress, IconButton, makeStyles, Typography } from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import { Cancel, CheckCircle } from '@material-ui/icons';
+import React, { FC, useEffect } from 'react';
+
+import PDFViewer from '../../../components/pdf/PDFViewer';
+import { setIsUploadingResume } from '../../../redux/substores/student/slices/resumeReviewSlice';
+import { resetUploadResume, setDocument } from '../../../redux/substores/student/slices/uploadResumeSlice';
+import { useStudentDispatch, useStudentSelector } from '../../../redux/substores/student/studentHooks';
+import { StudentDispatch } from '../../../redux/substores/student/studentStore';
+import initiateResumeReview from '../../../redux/substores/student/thunks/initiateResumeReview';
+import { arrayBufferToBase64, base64ToArrayBuffer } from '../../../util/helpers';
+
+const handleOnFileSelected = async (dispatch: StudentDispatch, files?: FileList | null) => {
+    // TODO: Improve error handling
+    if (files === undefined || files === null) {
+        alert('No file selected');
```

Comment:
I agree with using alerts. Simple and straightforward for the limited time we have.

Reply:
Yea, pop ups are will kick us sooner than later 😂